src/components/ReduxFormCreatableSelect/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AsyncCreatableSelect from 'react-select/async-creatable';
+
+import DiscoveryDataApiService from '../../data/services/DiscoveryDataApiService';
+
+const ReduxFormCreatableSelect = props => {
+  const {
+    input, defaultOptions, currValue, setValue,
+  } = props;
+
+  const loadOptions = (inputValue, callback) => DiscoveryDataApiService.fetchCourseTags(inputValue)
```

Comment:
Is this a specific format by AsyncCreatableSelect?

Reply:
yes